app/components/provider_interface/safeguarding_declaration_component.rb

```diff
@@ -8,12 +8,8 @@ def initialize(application_form:)
       @application_form = application_form
     end
 
-    def render?
-      application_form.disclose_disability? && disability_disclosure.present?
-    end
-
-    def disability_disclosure
-      @application_form.disability_disclosure
+    def has_disclosed_safeguarding_issues?
+      @application_form.safeguarding_issues.eql?('Yes') ? true : false
```

Comment:
This logic is 75% there, below is the form on the candidate side:

![image](https://user-images.githubusercontent.com/42817036/77336126-ff6cfa80-6d1e-11ea-8cf3-611770bcb77b.png)

A candidate can:

- choose 'Yes' which means `application_form.safeguarding_issues` => `Yes` 
- choose 'No' which means `application_form.safeguarding_issues` => `No` 
- choose 'Yes' and then add details e.g. 'foo' which means `application_form.safeguarding_issues` => `foo` 

So I believe the logic should be:

```suggestion
    def has_disclosed_safeguarding_issues?
      @application_form.safeguarding_issues != 'No'
```

Reply:
👍